code/controllers/subsystem/SSjobs.dm

```diff
@@ -521,7 +529,11 @@ SUBSYSTEM_DEF(jobs)
 				if(istype(G) && !G.prescription)
 					G.upgrade_prescription()
 					H.update_nearsighted_effects()
-
+	if(late_arrivals_spawning)
+		H.forceMove(pick(GLOB.latejoin))
+	if(!joined_late && drunken_spawning)
+		H.Sleeping(5 SECONDS)
+		H.Drunk(2 MINUTES)
```

Comment:
How much is this going to fuck over skrells?

Reply:
Livers tie to alchohol, not the status effect-... oh. Nope, nevermind, god damn it

Reply:
adjustment made to keep skrell mains / slime mains from killing me